Makefile

```diff
@@ -11,7 +11,7 @@ help:
 	@echo "dev - install theblues in develop mode"
 	@echo "lint - check style with flake8"
 	@echo "test - run tests quickly with the default Python"
-	@echo "check - run tests against 2.7 and 3.4 and check lint."
+	@echo "check - clean env, run tests against 2.7 and 3.4, check lint,"
 	@echo "test-all - run tests against 2.7 and 3.4, check lint, and test docs."
 	@echo "coverage - check code coverage quickly with the default Python"
```

Comment:
That is indeed a mismatch--thanks for spotting it. I've renamed the target to "coverage", to match targets like "check", "lint", etc.